android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -616,6 +607,28 @@ public void disableNativeFramesTracking() {
         }
     }
 
+    public void fetchNativeDeviceContexts(Promise promise) {
+        final Scope currentScope = InternalSentrySdk.getCurrentScope();
+        final Map<String, Object> serialized = InternalSentrySdk.serializeScope(
+                this.getReactApplicationContext().getApplicationContext(),
+                (SentryAndroidOptions) HubAdapter.getInstance().getOptions(),
```

Comment:
Can `getOptions` return `SentryOptions` instead? in case the SDK is NoOp, if so, we have to check the instance type beforehand.